etsd/keys/views.py

```diff
@@ -100,6 +72,31 @@ def form_valid(self, form):
         )
         return HttpResponseRedirect(reverse("home"))
 
+class PublicKeyAcceptRejectFormView(UpdateView):
+    model = models.PublicKey
+    form_class = forms.PublicKeyAcceptRejectForm
```

Comment:
Declare that this method only accepts `POST` (use http_method_names : https://ccbv.co.uk/projects/Django/3.1/django.views.generic.base/View/)